templates/client.hcl.j2

```diff
@@ -36,6 +36,18 @@ client {
 
 plugin "docker" {
   config {
+
+{% if nomad_docker_logging is defined %}
+  logging {
+    type = "{{ nomad_docker_logging['type'] }}"
+    config {
+      {% for key , value in nomad_docker_logging['config'].items() %}
```

Comment:
@tundeaoni This breaks indentation in the config file. The jinja tag should not have spaces at the start of the line